### PR TITLE
Reconstruction scripts w/ documentation

### LINKED ISSUE #308: 0.2.0's offline mode has dropped support for mixed-channel reconstructions
On Friday afternoon @Soorya19Pradeep and I installed `recOrder==0.2.0` on hummingbird and found that the removal of the fluorescence features from `recOrder` also removed support for reconstructions when some channels are polarization channels and other channels are fluorescence channels. This regression was unintended and I take responsibility. 

@mattersoflight, @ieivanov, @ziw-liu, and @edyoshikun discussed our plan yesterday, and we decided that we will remove the undertested and buggy offline mode from `recOrder` for our 0.3.0 release and migrate our current offline mode users to `waveorder` scripts where we've been performing mixed-channel reconstructions regularly. 

@Soorya19Pradeep I will support your transition to `waveorder` scripts.

### PR DESCRIPTION
**Pull request by talonchandler**
This PR reinstates a minimal set of example reconstruction scripts that users can modify for "QLIPP" and "PhaseFromBF" reconstructions. These scripts serve as a stopgap between `0.2.0`'s "Offline Mode" and `1.0.0`'s plan for a single autogenerated reconstruction mode. 

This PR also includes improved documentation for the plugin and the status of automated reconstructions (see the new `/docs/reconstruction-guide.md`). 

I will leave this as a draft, test on hummingbird tomorrow, and request review soon afterwards.

### CODE REVIEW

File: docs/napari-plugin-guide.md [hunk 1]
```diff
@@ -12,14 +11,12 @@ Launch `napari` with `recOrder`
 napari -w recOrder-napari
 ```
 
-## `recOrder` Calibration
-
-The first step in the QLIPP process is to calibrate the universal polarizer. This process involves generating the polarization states and acquiring a background **on an empty FOV**.  The light path must also be in **Kohler Illumination** in order to ensure uniform illumination of the sample. Steps on Kohler illumination can be found here https://www.microscopyu.com/tutorials/kohler.
+## Calibration tab
+The first step in the QLIPP process is to calibrate the liquid crystals. The goal is to illumination to  the contrast and
```
Comment (ziw-liu) [inline docs/napari-plugin-guide.md:15]: Accidentally truncated sentence?

**Comment by ziw-liu** [comment]
Linking #230.

File: examples/recon-phase-from-bf.py [hunk 2]
```diff
@@ -0,0 +1,68 @@
+from waveorder.io.reader import WaveorderReader
+from waveorder.io.writer import WaveorderWriter
+from recOrder.io.utils import load_bg
```
Comment (ziw-liu) [inline examples/recon-phase-from-bf.py:3]: Unused import. Does 'Phase from BF' use background?

**Comment by ziw-liu** [comment]
Since we are making a 100% diff change to `./examples`, maybe it's time to remove the now obsolete `./examples/__init__.py`? The new scripts are only supposed to be run 'as is' or copy-paste-modified, and does not define any importable object.

File: examples/recon-phase-from-bf.py (hunk 2 above)
Comment (talonchandler) [reply examples/recon-phase-from-bf.py]: Thanks!

File: docs/napari-plugin-guide.md (hunk 1 above)
Comment (talonchandler) [reply docs/napari-plugin-guide.md]: Thanks!

File: docs/napari-plugin-guide.md [hunk 3]
```diff
@@ -160,44 +93,43 @@ User specifies the directory in which the images will be saved. Needed if the *S
...
 Ex. for a 20 um thick cell, the user would first focus in the middle of the cell and then set the following parameters: 
 
-​									`Z Start: 	-12` 
-
-​									 `Z End:	 12`
-
-​									`Z Step:	 0.25` 
+​* `Z Start = -12`
+​* `Z End: 12`
+* `Z Step: 0.25`
```
Comment (ziw-liu) [inline docs/napari-plugin-guide.md:98]: Line breaks here may not [render](https://github.com/mehta-lab/recOrder/blob/87efe8d693c19b7709cedf97dfb0201b3552552d/docs/napari-plugin-guide.md) as intended

**Comment by ziw-liu** [comment]
This PR may also fix #242.

File: docs/napari-plugin-guide.md (hunk 3 above)
Comment (talonchandler) [reply docs/napari-plugin-guide.md:98]: Thanks!

File: docs/napari-plugin-guide.md [hunk 4]
```diff
@@ -12,145 +11,79 @@ Launch `napari` with `recOrder`
...
-
-[![DOI](https://zenodo.org/badge/DOI/10.5281/zenodo.5178487.svg)](https://doi.org/10.5281/zenodo.5178487)
-[![DOI](https://zenodo.org/badge/DOI/10.5281/zenodo.5135889.svg)](https://doi.org/10.5281/zenodo.5135889)
-
-Turn on online mode by clicking "Switch To Online" button.
+## Calibration tab
+The first step in the QLIPP process is to calibrate the liquid crystals. This process involves generating the polarization states and acquiring a background **on an empty FOV**, so begin by placing your sample on the stage and focussing on the surface of the coverslip or well. 
 
-![](https://github.com/mehta-lab/recOrder/blob/main/docs/images/switch_online.png)
+The light path must also be in Kohler Illumination** to ensure uniform illumination of the sample. [Please follow these steps to setup Kohler illumination.](https://www.microscopyu.com/tutorials/kohler)
```
Comment (ziw-liu) [inline docs/napari-plugin-guide.md:17]: Does the `**` indicate a mismatched **bold** or a footnote?

Comment (talonchandler) [reply docs/napari-plugin-guide.md]: Good catch!

File: docs/napari-plugin-guide.md [hunk 5]
```diff
@@ -12,145 +11,79 @@ Launch `napari` with `recOrder`
...
-The first step in the QLIPP process is to calibrate the universal polarizer. This process involves generating the polarization states and acquiring a background **on an empty FOV**.  The light path must also be in **Kohler Illumination** in order to ensure uniform illumination of the sample. Steps on Kohler illumination can be found here https://www.microscopyu.com/tutorials/kohler.
-
-More info on the details of the calibration process and use of the plugin can be found on Zenodo which were taken from a workshop at the Chan-Zuckerberg Biohub in June 2021.
-
-[![DOI](https://zenodo.org/badge/DOI/10.5281/zenodo.5178487.svg)](https://doi.org/10.5281/zenodo.5178487)
-[![DOI](https://zenodo.org/badge/DOI/10.5281/zenodo.5135889.svg)](https://doi.org/10.5281/zenodo.5135889)
-
-Turn on online mode by clicking "Switch To Online" button.
+## Calibration tab
+The first step in the QLIPP process is to calibrate the liquid crystals. This process involves generating the polarization states and acquiring a background **on an empty FOV**, so begin by placing your sample on the stage and focussing on the surface of the coverslip or well.
```
Comment (ziw-liu) [inline docs/napari-plugin-guide.md:15]: Not an error, but ['focusing'](https://books.google.com/ngrams/graph?content=focusing%2C+focussing&year_start=1900&year_end=2019&corpus=26&smoothing=3) seems to be more common.

Comment (talonchandler) [reply docs/napari-plugin-guide.md]: imma commonwealth boy!!!

File: docs/napari-plugin-guide.md [hunk 6]
```diff
@@ -12,145 +11,79 @@ Launch `napari` with `recOrder`
...
-![](https://github.com/mehta-lab/recOrder/blob/main/docs/images/switch_online.png)
+The light path must also be in **Kohler Illumination** to ensure uniform illumination of the sample. [Please follow these steps to setup Kohler illumination.](https://www.microscopyu.com/tutorials/kohler)
 
 After pressing “Connect to MM” and choosing a directory (where the calibration metadata / background images will be stored), the first step in calibration is to input the illumination wavelength and decide on the swing to use. 
 
 ![](https://github.com/mehta-lab/recOrder/blob/main/docs/images/connect_to_mm.png)
 
-### Deciding Swing
+### Choose a swing value
+The swing value is the deviation away from circular illumination states. The larger the swing, the more elliptical the polarized states becomes, until finally reaching a linear states at `swing = 0.25`. Picking a swing is dependent on the anisotropy of the sample. We recommend
```
Comment (ziw-liu) [inline docs/napari-plugin-guide.md:24]: <del>`a`</del>` linear states` or `a linear state`<del>`s`</del>?

File: docs/napari-plugin-guide.md [hunk 7]
```diff
@@ -12,145 +11,79 @@ Launch `napari` with `recOrder`
...
 ![](https://github.com/mehta-lab/recOrder/blob/main/docs/images/cap_bg.png)
 
 Choose the name of the folder to which to save your background images (will be placed into the save directory chosen at the beginning). Choose the number of images to average, 20 and below is generally good.  The background image results will then be displayed in the napari window.  It is normal to see some level of background retardance and orientation bias-- this can be corrected with the background correction step in reconstruction.  Examples of this display are below.
 
-*NOTE: If you wish the capture multiple background sets, please change the folder name in between captures as specifying the same name will overwrite previous data.*
-
 ![](https://github.com/mehta-lab/recOrder/blob/main/docs/images/bg_example.png)
 
 ### Advanced Tab
-
```
Comment (ziw-liu) [inline docs/napari-plugin-guide.md:131]: Wrapping headings with line breaks is recommended by [markdownlint](https://github.com/DavidAnson/markdownlint/blob/v0.26.2/doc/Rules.md#md022). But the parser that has issues with this does not comply to CommonMark specs, which GitHub adopted.

File: docs/napari-plugin-guide.md [hunk 8]
```diff
@@ -160,44 +93,43 @@ User specifies the directory in which the images will be saved. Needed if the *S
...
-![](https://github.com/mehta-lab/recOrder/blob/main/docs/images/reconstruct_acq2.png)
-
 Explanation of background correction methods:
-
-> <u>*None:*</u> No Background correction is performed. Not necessary to specify a background folder.
-
-> <u>*Global:*</u> A Global background correction is performed on every image by subtracting the background images from the sample in the stokes space.
-
-> <u>*Local Fit**:*</u> A global background correction is formed and an additional estimation of local background is computed with a polynomial surface fit.  This is the preferred background correction method.
+   bg_options = ["None", "Measured", "Estimated", "Measured + Estimated"]
```
Comment (ziw-liu) [inline docs/napari-plugin-guide.md:108]: Missing fences for code block?

**Review by ziw-liu — APPROVED** [review]

**Pull request merged**